### ISSUE TITLE
Folder structure can't cross platform   

### ISSUE THREAD

**Issue by HushmKun** [HushmKun/Artec#8]
Due to differences between NT Paths & POSIX Paths.

Since this works on windows, It won't follow to UNIX :
```
[
    {"folder": "src"},
    {"file": "src\\__init__.py"},
    {"folder": "test"},
    {"file": "test\\__init__.py"},
    {"folder": "res"},
    {"file": "README.md"},
    {"file": "setup.py"},
    {"file": "setup.cfg"},
    {"file": "pyproject.toml"},
]
``` 
If we wanna work it out on UNIX, It should be : 
```
[
    {"folder": "src"},
    {"file": "src/__init__.py"},
    {"folder": "test"},
    {"file": "test/__init__.py"},
    {"folder": "res"},
    {"file": "README.md"},
    {"file": "setup.py"},
    {"file": "setup.cfg"},
    {"file": "pyproject.toml"},
]
``` 
But, of course It won't work on windows.
So we have to either find a common ground or figure a new way.

**Comment by augustus-thomas** [comment]
Hey.

I found a module that might really help us. It is called pathlib. https://docs.python.org/3/library/pathlib.html

I think I might try to refactor my branch to use path objects (when I have time). I'm thinking I can try to convert from JSON to a path straight away. 

Sorry I haven't been contributing! I have a full time job right now (not is SWE) and so I do this in spare time or while procrastinating

**Comment by HushmKun** [comment]
> I found a module that might really help us. It is called pathlib. https://docs.python.org/3/library/pathlib.html

Thanks, @augustus-thomas.
I really did use the Pathlib Module. 😂

This issue is fixed as of 0.1.4
